aten/src/TH/generic/THTensorMath.h

```diff
@@ -22,6 +22,7 @@ TH_API void THTensor_(scatterFill)(THTensor *tensor, int dim, THLongTensor *inde
 
 TH_API accreal THTensor_(dot)(THTensor *t, THTensor *src);
 
+TH_API scalar_t THTensor_(powOne)(scalar_t x, scalar_t y);
```

Comment:
That's because you didn't move `powOne` before the first usage in THTensorMath.cpp.  It's not used outside of that file, it doesn't need to be exposed in a .h file.